### ISSUE TITLE
Accessibility issue with <List>

### ISSUE THREAD

**Issue by JWandrocke** [SAP/ui5-webcomponents#2371]
Missing ui type information when read by JAWS.

Created a List with SingleSelect. using the Arrow key to navigate up and done the list items.

Expected some like
List Item . **Is Active** .'item text' ‑ 
1 of ###

The Is Active is not read.

Using "@ui5/webcomponents": "1.0.0-rc.8",

**Comment by ilhan007** [comment]
Hello @JWandrocke

could you just confirm that it reproduces with 1.0.0-rc.9 before we start processing the issue?
Also, do you use the latest JAWS version and latest Chrome?

And, can you check if you the expected reading is in the openui5 equivalent control - sap.m.List
https://openui5.hana.ondemand.com/entity/sap.m.List/sample/sap.m.sample.ListSelection
The dropdown at the top has selection options, so you can pick the "SingleSelect" one.

Best,
ilhan

**Comment by JWandrocke** [comment]
@ilhan007 Just tried 1.0.0-rc.9 and the problem still occurs.  The version of JAWS (2018 1811.30 ILM) is up to date per our https://wiki.wdf.sap.corp/wiki/display/Acc/How+to+Install+and+Use+Freedom+Scientific+Jaws page
And Chrome is the latest also (86.0.4240.111).

Only hear

> List Item
> My Text
> 1 of 5


No indications that it can be click/pressed.

code is as follows:

```
          <List
            tabIndex={tabIndex}
            className={cx('caiList', { caiFocusIndicator: tabIndex })}
            mode='SingleSelect' onItemClick={this.doSendMessage}>
            {liItems}
          </List>

```
I can send you the actual BCP from our QA team if that will help.

Regards,
John.wandrocke@sap.com
SAP Labs, Palo Alto

**Comment by jyguyomarch** [comment]
@ilhan007 any progress here?

**Comment by dobrinyonkov** [comment]
Hello @jyguyomarch,
Hello @JWandrocke,

This is a known issue with JAWS and the “aria-selected” attribute.
There is an open issue marked as JAWS Bug by FreedomScientific, regarding this problem.

See: [FreedomScientific/VFO-standards-support#440](https://github.com/FreedomScientific/VFO-standards-support/issues/440).

Kind Regards,
Dobrin
